tasks/steps/cleanup_instance.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: 'Check that instance is not started'
+  cartridge_check_instance_not_started:
+    console_sock: '{{ instance_info.console_sock }}'
+  register: check_instance_not_started_res
+
+- name: 'Cleanup instance files'
+  file:
+    path: '{{ file_path }}'
+    state: absent
+  loop_control:
+    loop_var: file_path
+  with_items: '{{ instance_info.files_to_remove_on_cleanup }}'
```

Comment:
`cartridge_cleanup_excluded_paths` isn't applied here.

Reply:
I'm not sure that it's good to allow the user to ignore PID and socket